.travis.yml

```diff
@@ -1,21 +1,21 @@
-language: python
-sudo: false
+language: go
+
 env:
-  - TOX_ENV=py27
-  - TOX_ENV=py35
-install:
-  - cd python
-  - pip install tox
-  - pip install .
-script: tox -e $TOX_ENV
-after_success:
-  - if [[ "${TOX_ENV}" == "py27" ]]; then tox -e coveralls; fi
+  - GO111MODULE=on
+
+go:
+  - 1.12.x
```

Comment:
Do we want to cover multiple versions?

Reply:
Adding 1.11.x. 1.13.x is not yet available.